src/Component/tests/spec/Doctrine/Persistence/Exception/ResourceExistsExceptionSpec.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace spec\Sylius\Resource\Doctrine\Persistence\Exception;
+
+use PhpSpec\ObjectBehavior;
+
+final class ResourceExistsExceptionSpec extends ObjectBehavior
+{
+    function it_extends_exception(): void
+    {
+        $this->shouldHaveType(\Exception::class);
```

Comment:
Should it be more specific?
```suggestion
        $this->shouldHaveType(\RuntimeException::class);
```

Reply:
I added the two assertions, cause the moved one was extending exception one.